src/internal/connector/onedrive/item.go

```diff
@@ -191,12 +191,24 @@ func oneDriveItemMetaInfo(
 
 	perm, err := service.Client().DrivesById(driveID).ItemsById(*itemID).Permissions().Get(ctx, nil)
 	if err != nil {
-		return Metadata{}, errors.Wrapf(err, "failed to get item permissions %s", *itemID)
```

Comment:
since this error isn't wrapped elsewhere please keep at least the part that says we failed fetching the permissions. I'm unsure if we need/should keep the item ID

Reply:
This was done so that the errors returned from the client are passed as is to the checks [here](https://github.com/alcionai/corso/blob/18a8c350f7e9e5e12a3296c1b87e1d032841654d/src/internal/connector/onedrive/collection.go#L296). But let me wrap it before adding it to `errUpdater`.